api/tests/identity-access-management/integration/domain/usecases/register-lti-platform.test.js

```diff
@@ -0,0 +1,263 @@
+import { InvalidLtiPlatformRegistrationError } from '../../../../../src/identity-access-management/domain/errors.js';
+import { usecases } from '../../../../../src/identity-access-management/domain/usecases/index.js';
+import { ltiPlatformRegistrationRepository } from '../../../../../src/identity-access-management/infrastructure/repositories/lti-platform-registration.repository.js';
+import { catchErr, domainBuilder, expect, knex, sinon } from '../../../../test-helper.js';
+
+describe('Integration | Identity Access Management | Domain | Usecases | register-lti-platform', function () {
+  it('should discover platform’s open ID configuration and register Pix configuration on the platform', async function () {
```

Comment:
Pour les titres des tests nous utilisons des titres « assertifs » plutôt que « déclaratifs » : 

```suggestion
  it('discovers platform’s open ID configuration and register Pix configuration on the platform', async function () {
```